packages/cms-cli/bin/cli.js

```diff
@@ -57,13 +57,11 @@ const argv = yargs
   .command(filemanagerCommand)
   .command(secretsCommand)
   .help()
-  .demandCommand(
-    1,
-    `Please specify a command or run \`${yargs.$0} --help\` for a list of available commands`
-  )
   .recommendCommands()
   .strict().argv;
 
 if (argv.help) {
   trackHelpUsage(getCommandName(argv));
+} else if (argv['_'].length == 0) {
+  yargs.showHelp();
```

Comment:
Makes sense. I wonder if we should also include some copy to indicate that a command is required sort of like the message that was used previously.

Reply:
As I was working on getting a message added, I might have found a less hacky way of showing help. Will have that up shortly

Reply:
Never mind, I was going to add `showHelp()` to the `.fail()` cb, but it looks like we already went down that road in https://github.com/HubSpot/hubspot-cms-tools/pull/305#pullrequestreview-478853884